### PR TITLE
Doubles PACMAN power output, triples their sheet efficiency. Inducers can now be recharged with plasma.

### PR DESCRIPTION
**Pull request by SmArtKar**
## About The Pull Request

PACMANs now can produce 10-40kW of power and take 180 ticks to go through a sheet instead of 60. This does not allow them to power the station on their own like in the good ole days, but they're still capable of powering a section of a department(maybe even all of it if it doesn't eat too much power)

Inducers can be recharged with plasma, 15kJ per sheet. Cargo-ordered inducers now also start with an upgraded megacell (x2.5 of a normal one) instead of a regular upgraded battery (x5 of a normal one and half of a megacell) (it barely can charge anything)

## Why It's Good For The Game

PACMANs are currently in a very sad state as they are borderline useless after power changes, and don't see much use beyond kickstarting the SM if emitters don't have enough juice. This should bring them a bit into action while not allowing them to power the entire station like before.
As for emitter changes, introduction of megacells rendered them almost entirely useless as you cannot charge megacells you print and using normal batteries for charging APCs is a joke. This should allow people to recharge their APCs and machinery in a pinch. Inducer change has been approved by Watermelon a while ago, while PACMAN buffs have been mentioned but these are significantly smaller than discussed with him.

## Changelog
:cl:
balance: PACMANs now have significantly increased power output and take longer to consume a single sheet
balance: Inducers can now be recharged with plasma
balance: Inducers ordered from cargo now start with upgraded megacells instead of upgraded batteries
/:cl:

### CODE REVIEW

File: code/modules/power/port_gen.dm [hunk 1]
```diff
@@ -109,8 +109,8 @@
 		icon_state = "portgen1_0"
 		base_icon_state = "portgen1"
 		max_sheets = 20
-		time_per_sheet = 20
-		power_gen = 15000
+		time_per_sheet = 60
+		power_gen = 30000
```
Comment (SyncIt21) [inline code/modules/power/port_gen.dm:113]: Cell defines and same everywhere else

**Review by SyncIt21 — CHANGES_REQUESTED** [review]
Need to use defines. No magic numbers

File: code/modules/power/port_gen.dm [hunk 2]
```diff
@@ -82,13 +82,13 @@
 /obj/machinery/power/port_gen/pacman
 	name = "\improper P.A.C.M.A.N.-type portable generator"
 	circuit = /obj/item/circuitboard/machine/pacman
-	power_gen = 5000
+	power_gen = 10000
```
[the remark this answers is not in the document]
Comment (Pickle-Coding) [reply code/modules/power/port_gen.dm:85]: Doesn't make sense to use cell defines. The pacman is used to power a portion of the station, which is mostly independant of cell capacity. Could just use the metric units.

File: code/modules/power/port_gen.dm [hunk 3]
```diff
@@ -82,13 +82,13 @@
...
 	var/sheets = 0
 	var/max_sheets = 50
 	var/sheet_name = ""
 	var/sheet_path = /obj/item/stack/sheet/mineral/plasma
 	var/sheet_left = 0 // How much is left of the sheet
-	var/time_per_sheet = 60
+	var/time_per_sheet = 180
```
[the remark this answers is not in the document]
Comment (SmArtKar) [reply code/modules/power/port_gen.dm:91]: Its SSmachines process cycles, so double that in seconds.

File: code/modules/power/port_gen.dm (hunk 2 above)
[the remark this answers is not in the document]
Comment (SmArtKar) [reply code/modules/power/port_gen.dm:85]: I agree with pickle, this has nothing to do with cells. I will convert it to kJ and thats all

**Comment by Loafin34** [comment]
RETURN OF THE M.R.S. P.A.C.M.A.N.

**Pull request merged**